kas-docker.yml

```diff
@@ -0,0 +1,9 @@
+header:
+  version: 8
+  includes: 
+  - kas.yml
+
+distro: pce-base
+machine: pcengines-apu2
+target:
+  - docker-image-minimal
```

Comment:
@Asiderr It would be nice to use / configure editor in a way that the newlines are correctly added at the EOF.